test/unit/pj_phi2_test.cpp

```diff
@@ -39,47 +39,63 @@ namespace {
 TEST(PjPhi2Test, Basic) {
     projCtx ctx = pj_get_default_ctx();
 
-    EXPECT_DOUBLE_EQ(M_PI_2, pj_phi2(ctx, 0.0, 0.0));
-
-    EXPECT_NEAR(0.0, pj_phi2(ctx, 1.0, 0.0), 1e-16);
-    EXPECT_DOUBLE_EQ(M_PI_2, pj_phi2(ctx, 0.0, 1.0));
-    EXPECT_DOUBLE_EQ(M_PI, pj_phi2(ctx, -1.0, 0.0));
-    EXPECT_DOUBLE_EQ(M_PI_2, pj_phi2(ctx, 0.0, -1.0));
-
-    EXPECT_NEAR(0.0, pj_phi2(ctx, 1.0, 1.0), 1e-16);
-    EXPECT_DOUBLE_EQ(M_PI, pj_phi2(ctx, -1.0, -1.0));
-
-    // TODO(schwehr): M_PI_4, M_PI_2, M_PI, M_E
-    // https://www.gnu.org/software/libc/manual/html_node/Mathematical-Constants.html
-
-    EXPECT_DOUBLE_EQ(-0.95445818456292697, pj_phi2(ctx, M_PI, 0.0));
-    EXPECT_TRUE(std::isnan(pj_phi2(ctx, 0.0, M_PI)));
-    EXPECT_DOUBLE_EQ(4.0960508381527205, pj_phi2(ctx, -M_PI, 0.0));
-    EXPECT_TRUE(std::isnan(pj_phi2(ctx, 0.0, -M_PI)));
-
-    EXPECT_TRUE(std::isnan(pj_phi2(ctx, M_PI, M_PI)));
-    EXPECT_TRUE(std::isnan(pj_phi2(ctx, -M_PI, -M_PI)));
+    // Remove tests with |e| >= 1.  Expectation is that only sane values of e
+    // (and nan is here reckoned to be sane) are passed to pj_phi2.  Thus the
+    // return value with other values of e is "implementation dependent".
```

Comment:
This is great commentry in the PR but is mostly confusion once merged. Remove or rephrase to "We don't test cases with |e| >= 1, because the expectation is ..."

Reply:
Fixed